deps/compiler-rt.mk

```diff
@@ -0,0 +1,144 @@
+##
+# In order to support fallbacks within llvm we need to support
+# compiler-rt. This means linking sys.so against it and resolving
+# symbols at during JIT. For the latter part we need to create
+# a .so that we can load, but compiler-rt only comes in a .a.
+#
+# There are several configurations to take into account.
+# 1. USE_SYSTEM_LLVM == 1
+#    libclang_rt.builtins is distributed with clang and so
+#    we assume that USE_SYSTEM_LLVM == 1 means that clang is also
```

Comment:
At least for Fedora RPMs, I need  `USE_SYSTEM_LLVM=1` and I also have to build LLVM with gcc. Is that a problem, or is installing clang enough?